tests/unittests/audio/test_c_si_snr.py

```diff
@@ -0,0 +1,97 @@
+# Copyright The Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from collections import namedtuple
+
+import pytest
+import torch
+from scipy.io import wavfile
+from torchmetrics.audio import ComplexScaleInvariantSignalNoiseRatio
+from torchmetrics.functional.audio import complex_scale_invariant_signal_noise_ratio
+
+from unittests import BATCH_SIZE, NUM_BATCHES
+from unittests.audio import _SAMPLE_AUDIO_SPEECH, _SAMPLE_AUDIO_SPEECH_BAB_DB
+from unittests.helpers import seed_all
+from unittests.helpers.testers import MetricTester
+
+seed_all(42)
+
+Input = namedtuple("Input", ["preds", "target"])
+
+inputs = Input(
+    preds=torch.rand(NUM_BATCHES, BATCH_SIZE, 129, 20, 2),
+    target=torch.rand(NUM_BATCHES, BATCH_SIZE, 129, 20, 2),
+)
+
+
+@pytest.mark.parametrize(
+    "preds, target, ref_metric, zero_mean",
+    [
+        (inputs.preds, inputs.target, None, True),
+        (inputs.preds, inputs.target, None, False),
+    ],
+)
+class TestComplexSISNR(MetricTester):
```

Comment:
We don't test our metric against any reference one? 🤔

Reply:
It was discussed at some point in the slack channel that for metrics that does not have a reference implementation we settle for some fixed test cases (see the tests below the test class)